cryptography/src/jvmMain/kotlin/com/wire/kalium/cryptography/ProteusClientCryptoBoxImpl.kt

```diff
@@ -84,6 +84,10 @@ class ProteusClientCryptoBoxImpl constructor(
         return wrapException { box.localFingerprint }
     }
 
+    override suspend fun remoteFingerPrint(sessionId: CryptoSessionId): ByteArray {
+        TODO("Not yet implemented")
```

Comment:
We can implement this in https://github.com/wireapp/cryptobox4j.

But at the same time... We're dropping Cryptobox for CoreCrypto anyway, so I guess it's fine to leave it like this.